style.css

```diff
@@ -0,0 +1,101 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #f5f5f5;
+}
+
+.container {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  padding: 36px;
+  color: #fff;
+  background-color: #3c3a39;
+  height: 92px;
+}
+
+.logo {
+  color: #fff;
+  font-weight: 700;
+  font-family: 'Poppins', sans-serif;
+  height: 24px;
+  text-decoration: none;
+}
+
+.section {
+  display: flex;
+  background-color: #1c1a19;
+  background-image: url(./images/Colorcard-hex_1c1a19-www.colorbook.io.png);
+  background-repeat: no-repeat;
+  background-position: right bottom;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 110px 24px;
+  width: 100%;
+  height: 610px;
+  color: #fff;
+  gap: 12px;
+}
+
+.main-title {
+  height: 124px;
+  color: #ff6b00;
+  font-size: 60px;
+  line-height: 67px;
```

Comment:
The original font size is **56** pixels, and the line height is **64** pixels. Kindly make sure to fix it.